src/Common/src/System/SR.cs

```diff
@@ -81,4 +81,4 @@ internal static string Format(string resourceFormat, object p1, object p2, objec
             return string.Format(resourceFormat, p1, p2, p3);
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
NIT: we can undo this change 😄

Reply:
That was automatically made by some tool :smile: I will undo it.